src/tactic/spass/examples.lean

```diff
@@ -0,0 +1,115 @@
+/-
+  Copyright (c) 2019 Seul Baek. All rights reserved.
+  Released under Apache 2.0 license as described in the file LICENSE.
+  Author: Seul Baek
+-/
+
+import tactic.spass.main
+
+section
+
+/- Examples from finish. -/
+
+variables {A B C D : Prop}
+
+  example : ¬ A → A → C := by spass
+  example : (A ∧ A ∧ B) → (A ∧ C ∧ B) → A := by spass
+  example : A → ¬ B → ¬ (A → B) := by spass
+  example : A ∨ B → B ∨ A := by spass
+  example : A ∧ B → B ∧ A := by spass
+  example : A → (A → B) → (B → C) → C := by spass
+  example : (A ∧ B) → (C ∧ B) → C := by spass
+  example : A → B → C → D → (A ∧ B) ∧ (C ∧ D) := by spass
+  example : (A ∧ B) → (B ∧ ¬ C) → A ∨ C := by spass
+  example : (A → B) ∧ (B → C) → A → C := by spass
+  example : (A → B) ∨ (B → A) := by spass
+  example : ((A → B) → A) → A := by spass
+  example : A → ¬ B → ¬ (A → B) := by spass
+  example : ¬ (A ↔ ¬ A) := by spass
+  example : (A ↔ B) → (A ∧ B → C) → (¬ A ∧ ¬ B → C) → C := by spass
+  example : (A ↔ B) → ((A ∧ ¬ B) ∨ (¬ A ∧ B)) → C := by spass
+  example : (A → B) → A → B := by spass
+  example : (A → B) → (B → C) → A → C := by spass
+  example : (A → B ∨ C) → (B → D) → (C → D) → A → D := by spass
+  example : A ∨ B → B ∨ A := by spass
+
+  variables (α : Type) [inhabited α]
+  variables (a b c : α) (p q : α → Prop) (r : α → α → Prop)
+  variables (P Q R : Prop)
+  variable  (g : bool → nat)
+
+  example : (∀ x, p x → q x) → (∀ x, p x) → q a := by spass
```

Comment:
I get an error here:
```
s0(U)
   ^

expected: s | u | v | w | y | z | x

state:
α : Type,
_inst_1 : inhabited α,
a : α,
p q : α → Prop
⊢ (∃ (x : α), p x ∧ ¬q x) ∨ (∃ (x : α), ¬p x) ∨ q a
```

Reply:
Perhaps the problem is that only SPASS 3.5 is available on Windows/Mac, and you're depending on a later version? If so, let's explain this in the README.